jobs/JRRFS_UNGRIB

```diff
@@ -42,6 +42,7 @@ export NAME_PATTERN=${NAME_PATTERN:-("")}
 export OFFSET=${OFFSET:-0}
 export LENGTH=${LENGTH:-0}
 export INTERVAL=${INTERVAL:-1}
+export SOURCE=${SOURCE:-"(GFS)"}
```

Comment:
Is `SOURCE` an array?

Reply:
No. Change to 
export SOURCE=${SOURCE:-"GFS"}